docs/runtime/workers.md

```diff
@@ -9,10 +9,10 @@ is run on a separate thread, dedicated only to that worker.
 Currently Deno supports only `module` type workers; thus it's essential to pass
 the `type: "module"` option when creating a new worker.
 
-Relative module specifiers are
-[not supported](https://github.com/denoland/deno/issues/5216) at the moment. You
-can instead use the `URL` constructor and `import.meta.url` to easily create a
-specifier for some nearby script.
+Relative module specifiers are only supported with `--location <href>` passed on
+the CLI. This is not recommended for portability. You can instead use the `URL`
+contructor and `import.meta.url` to easily create a specifier for some nearby
+script.
```

Comment:
This is true only for workers create from main worker. If I understand correctly all workers have location set to the resolved specifier, so for "2nd generation" workers one can always use relative module specifier.

Reply:
I think 2nd gen workers still should be relative to top level window.location right? I'd have to check the spec.